BlackbaudAddinHelloWorld/HelloWorldButton.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8" />
+	<title></title>
+	<script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/@blackbaud/sky-addin-client@1.0.2/bundles/sky-addin-client.umd.min.js"></script>
+	<script src="HelloWorldButton.js"></script>
+</head>
+<body>
+	<p>Hello world button</p>
```

Comment:
I don't think this is needed, right?  There is no UI from the HTML that is rendered, just used to get the script downloaded

Reply:
It's not necessary.